libs/dh/admin/feature-permissions/src/lib/overview/dh-admin-permission-overview.component.ts

```diff
@@ -97,7 +91,26 @@ export class DhAdminPermissionOverviewComponent implements OnInit, OnDestroy {
   }
 
   refreshData(): void {
-    this.getPermissionsQuery.refetch();
```

Comment:
suggestion: instead of unsubscribing and subscribing again, the new search term can be sent to `refetch` as a param. Have you tried that? See `dh-wholesale-search.component.ts#105` as an example. This way the original `getPermissionsWatchQuery` function will stay mostly the same (except the addition of `variables`).

Reply:
Fixed